samples/MicStream.js

```diff
@@ -0,0 +1,378 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic recognize operations with
+ * with the Google Cloud Speech API.
+ *
+ * For more information, see the README.md under /speech and the documentation
+ * at https://cloud.google.com/speech/docs.
+ */
+
+'use strict';
+
+/**
+ * Note: Correct microphone settings is required: check enclosed link, and make
+ * sure the following conditions are met:
+ * 1. SoX must be installed and available in your $PATH- it can be found here:
+ * http://sox.sourceforge.net/
+ * 2. Microphone must be working
+ * 3. Encoding, sampleRateHertz, and # of channels must match header of audio file you're
+ * recording to.
+ * 4. Get Node-Record-lpcm16 https://www.npmjs.com/package/node-record-lpcm16
+ * More Info: https://cloud.google.com/speech-to-text/docs/streaming-recognize
+ */
+
+//uncomment below and see Errata at end of file for more configurations
```

Comment:
We only use uncommented "placeholder variables" like this inside of `[START` `END]` "region tags" for embedding in the documentation

I'll follow up privately with a guide for sample style